packages/@react-spectrum/menu/src/MenuItem.tsx

```diff
@@ -96,9 +96,6 @@ export function MenuItem<T>(props: MenuItemProps<T>) {
     endProps.id = endId;
     menuItemProps['aria-describedby'] = [menuItemProps['aria-describedby'], endId].filter(Boolean).join(' ');
   }
-  if (isUnavailable) {
-    menuItemProps['aria-disabled'] = 'true';
-  }
```

Comment:
@majornista @jnurthen Any opinions on the announcement of the unavailable menu items here? Here is a previous version: https://reactspectrum.blob.core.windows.net/reactspectrum/17e72999fefe843a1261da0821132866b5fec3ca/storybook/index.html?path=/story/menutrigger--menu-item-unavailable&providerSwitcher-express=false. 

IMO perhaps we could add a custom label indicating that an item's default action unavailable but more info can be found

Reply:
We already have an `aria-describedby` pointing to the icon with the `aria-label` "Expand for more details": https://github.com/adobe/react-spectrum/blob/ec32f0c21773ad22fc0228882e0fc07122d5bac5/packages/%40react-spectrum/menu/intl/en-US.json#L3, perhaps we could add "Unavailable: " to the start of this string.

Reply:
Sounds good to me, I'll make the update